src/kakarot/accounts/contract/contract_account.cairo

```diff
@@ -58,13 +58,16 @@ func state{
     return ContractAccount.read_state(key);
 }
 
+// @notice This function is used to Initiate the smart contract.
```

Comment:
initialize*

although our problem is also true when talking about the method itself, it should be initialize, but we'll